src/m365/todo/commands/task/task-add.ts

```diff
@@ -64,26 +118,44 @@ class TodoTaskAddCommand extends GraphCommand {
     try {
       const listId: string = await this.getTodoListId(args);
 
-      const requestOptions: any = {
+      const requestOptions: AxiosRequestConfig = {
         url: `${endpoint}/me/todo/lists/${listId}/tasks`,
         headers: {
           accept: 'application/json;odata.metadata=none',
           'Content-Type': 'application/json'
         },
         data: {
-          title: args.options.title
+          title: args.options.title,
+          body: {
+            content: args.options.bodyContent,
+            contentType: args.options.bodyContentType?.toLowerCase() || 'text'
```

Comment:
bodyContentType doesn't seem to be applied to the body. See image below.

![afbeelding](https://user-images.githubusercontent.com/38426621/196459336-e478aacf-6ff2-45f2-a7ae-4ec4678c61dc.png)

Update: The UI seems to apply the HTML body correctly but the response doesn't output it correctly.

Reply:
Seems like this is also some API behavior 😊. Seems like the API translates the HTML to some sort of text markup.
Checking the executed request, the request body seems fine.